.circleci/config.yml

```diff
@@ -1,15 +1,9 @@
 defaults: &defaults
   working_directory: ~/ndelius-new-tech
   docker:
-    - image: circleci/openjdk:8-node-browsers
-
-install_aws_cli: &install_aws_cli
-  run:
-    name: Set up aws
-    command: |
-      sudo apt-get --assume-yes update
-      sudo apt-get --assume-yes install python3-pip
-      sudo pip3 install awscli
+    # dangerous pinning to old version with node v12.18.1 - from previous version of circleci/openjdk:8-node-browsers
+    # fixes issue with compatible issue with v12.18.3 and graceful-fs webjar
+    - image: circleci/openjdk@sha256:0ab7db4ecdc5966baaa44f64075725c22a53dc63922492c26cabd98b7c961ae0
 
```

Comment:
see comment, unrelated - but circle build is broken due to a JavaScript compile issue related to upgrade in the circleci node version. So this pins version back to the last time the build succeeded